tests/constants-e2e.ts

```diff
@@ -989,6 +997,8 @@ export const Holders: {
     nETH: '0xcf2ef00e75558512ae735679ea5df62ad2056786',
     avWETH: '0x92d78e32b990d10aeca0875dc5585f1a6f958179',
     YUSD: '0x6c1a5ef2acde1fd2fc68def440d2c1eb35bae24a',
+    EUROC: '0xC891EB4cbdEFf6e073e859e987815Ed1505c2ACD',
+    GMX: '0x62edc0692BD897D2295872a9FFCac5425011c661',
```

Comment:
Please remove L1000-L1001 as the generic-rfq e2e test uses Tenderly state overrides instead of the old "Holders" mechanism.

Reply:
ready @skypper !